#87: Updated user guide for added error message column (#89)
exasol/transformers-extension: doc/user_guide/user_guide.md

The inference results are presented with predicted _LABEL_ and confidence 
 _SCORE_ columns, combined with the inputs used when calling 
this UDF. In case of any error during model loading or prediction, these new 

> columns are set to `null`, and you can see the stacktrace of the error in the 
> _ERROR_MESSAGE_ column. For example:

Reviewer: ```suggestion
columns are set to `null` and column _ERROR_MESSAGE_ is set 
to the stacktrace of the error. For example:
```

  - ```second_text```: The second input text

The inference results are presented with predicted _LABEL_ and confidence 
 _SCORE_ columns, combined with the inputs used when calling this UDF. 
In case of any error during model loading or prediction, these new 

> columns are set to `null`, and you can see the stacktrace of the error in the 
> _ERROR_MESSAGE_ column.

Reviewer: ```suggestion
columns are set to `null` and column _ERROR_MESSAGE_ is set 
to the stacktrace of the error. 
```

The inference results are presented with predicted _ANSWER_, confidence 
 _SCORE_, and _RANK_ columns, combined with the inputs used when calling this UDF.
If `top_k` > 1, each input row is repeated for each answer. In case of any error 

> during model loading or prediction, these new columns are set to `null`, and 
> you can see the stacktrace of the error in the _ERROR_MESSAGE_ column. For example:

Reviewer: ```suggestion
during model loading or prediction, these new columns are set to `null` and column _ERROR_MESSAGE_ is set 
to the stacktrace of the error. For example:
```

The inference results are presented with _FILLED_TEXT_, confidence 
 _SCORE_, and _RANK_ columns, combined with the inputs used when calling this UDF.
If `top_k` > 1, each input row is repeated for each prediction. In case of any 

> error during model loading or prediction, these new columns are set to `null`, 
> and you can see the stacktrace of the error in the _ERROR_MESSAGE_ column. For example:

Reviewer: ```suggestion
error during model loading or prediction, these new columns are set to `null` 
and column _ERROR_MESSAGE_ is set to the stacktrace of the error. For example:
```

  - ```return_full_text```:  If set to False only added text is returned, otherwise the full text is returned.

The inference results are presented with _GENERATED_TEXT_ column, 
combined with the inputs used when calling this UDF. In case of any error during 
model loading or prediction, these new columns are set to `null`, and you can 

> see the stacktrace of the error in the _ERROR_MESSAGE_ column.

Reviewer: (see above)

  - ```max_length```: The maximum total length of the translated text. 

The inference results are presented with _TRANSLATION_TEXT_ column, 
combined with the inputs used when calling this UDF. In case of any error during
model loading or prediction, these new columns are set to `null`, and you can 

> see the stacktrace of the error in the _ERROR_MESSAGE_ column. For example:

Reviewer: (see above)

  should be comma-separated, e.g., `label1,label2,label3`.
  
The inference results are presented with predicted _LABEL_, _SCORE_ and _RANK_ 
columns, combined with the inputs used when calling this UDF. In case of any 
error during model loading or prediction, these new  columns are set to `null`, 

> and you can see the stacktrace of the error in the _ERROR_MESSAGE_ column. For example:

Reviewer: (see above)
